File: python/7 kyu/330.py
# https://www.codewars.com/kata/52f5424d0531259cfc000d04/train/python

# Passed

def sort_by_bit(seq):
    print(seq)
    seq = [idx for idx in seq if idx < 32]
    binary_num_arr = ["0"] * (max(seq) + 1) if len(seq) != 0 else ["0"] 
    for idx in seq:
        binary_num_arr[idx] = "1"
        
    binary_num_arr = binary_num_arr[::-1]
    binary_num = "".join(binary_num_arr)
    result = int(binary_num, 2)
    return result

output = sort_by_bit([1, 0])
print(output)

File: nodejs/7 kyu/330.js
// https://www.codewars.com/kata/537baa6f8f4b300b5900106c/train/javascript

// Passed

function circleArea(radius) {
  if (radius <= 0) {
    throw new Error("radius has to be > 0");
  }
  let area = Math.PI * radius ** 2;
  return area;
}

const output = circleArea(68);
console.log(output);